bots/bus-api/index.js

```diff
@@ -182,8 +178,8 @@ function checkpoint(request, context, callback) {
 		type: request.type
 	};
 	if (request.event) {
-		params.eid = event.eid || params.eid;
```

Comment:
"event" was undefined on this line
I figured it was supposed to be request.event.eid
This may have fixed a bug?